src/routes/living-form/containers/LivingFormPage.js

```diff
@@ -1,138 +1,59 @@
-/* eslint-disable */
 import React from 'react'
-import { Control, Errors, Fieldset, LocalForm } from 'react-redux-form'
-import { field } from './LivingFormPage.css'
 import getCurrentCityConfig from 'modules/city-detection/getCurrentCityConfig'
 import serverConfig from 'server.config'
 import { Caption } from '@integreat-app/shared'
+import NeuburgForm from './NeuburgForm'
+import Failure from '../../../modules/common/components/Failure'
 
 const cityConfig = getCurrentCityConfig()
-const Address = () => (
-  <Fieldset model=".address">
-    <div className={field}>
-      <label>Adresse:</label>
-      <Control.text model=".address" />
-    </div>
-    <div className={field}>
-      <label>City:</label>
-      <Control.text model=".city" />
-    </div>
-    <div className={field}>
-      <label>Bundesland:</label>
-      <Control.text model=".state" />
-    </div>
-    <div className={field}>
-      <label>Postleitzahl:</label>
-      <Control.text model=".zip" />
-    </div>
-  </Fieldset>
-)
+const STATUS_CREATED = 201
 
 export class LivingFormPage extends React.Component {
   constructor () {
     super()
-    this.state = {success: false}
+    this.state = {success: false, serverError: null, sending: false, emailAddress: ''}
   }
 
-  handleChange (values) {
-
-  }
-
-  handleUpdate (form) { }
-
-  handleSubmit (values) {
-    console.log(values)
-    const {landlord, property} = values
+  sendRequest = requestBody => {
+    console.log('Sending request with body:')
+    console.log(requestBody)
+    this.setState({sending: true, serverError: null, emailAddress: requestBody.email})
     fetch(`${serverConfig.host}/v0/${cityConfig.cmsName}/`, {
-      body: JSON.stringify({email: landlord.email, duration: property.duration * 24 * 60 * 60, formData: {}}),
+      body: JSON.stringify(requestBody),
       method: 'PUT',
       headers: new Headers({
         'Content-Type': 'application/json'
       })
-    }).then(response => response.status)
-      .then(status => this.setState({success: status === 200}))
+    }).then(response => {
+      if (response.status === STATUS_CREATED) {
+        this.setState({success: true, sending: false})
+      } else {
+        return response.text().then(text => this.setState({serverError: text, sending: false}))
+      }
+    })
   }
 
   render () {
-    if (this.state.success) {
-      return <Caption title={'Angebot wurde erstellt. Überprüfen Sie ihr E-Mail Postfach :)'} />
+    if (cityConfig.cmsName !== 'neuburgschrobenhausenwohnraum') {
```

Comment:
Is this hard-coded string really necessary?

Reply:
Hmm.. that was the easiest possibility I've found to determine, if the currentCity is neuburg^^ Any suggestion?

Reply:
We could refactor the cityConfigs to be an object instead of an array and then check whether the cityConfig (or the cmsName of the cityConfig) returned by getCurrentCityConfig is the same as the neuburg config (cmsName of the neuburgConfig)

Reply:
+1 for @klinzo's proposal.
For me it would be ok if you leave it like this until we have the city configs as an api service.